### PR TITLE
Prep info for election process including elekto

### PR DESCRIPTION
**Pull request by dims**
Add the election.yaml used by elekto and the nomination template as well for 2022 Steering Election

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: elections/steering/2022/election.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: 2022 Steering Committee Election
+organization: Kubernetes
+start_datetime: 2022-09-06 00:00:01
+end_datetime: 2022-09-29 23:59:00
```
Comment (dims) [inline elections/steering/2022/election.yaml:4]: @jberkus @coderanger I don't know how to specify the AoE time

Comment (dims) [reply elections/steering/2022/election.yaml:4]: https://www.timeanddate.com/worldclock/converter.html?iso=20220930T115900&p1=tz_aoe&p2=1440

Shows 11:59:59 on Sep 30, 2022. will go with that.

**Comment by coderanger** [comment]
/lgtm
/approve

Let's get this party started!

**Pull request merged**